src/wallet/wallet.cpp

```diff
@@ -2546,6 +2548,9 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend, CWalletTx& wt
     // all.
     txNew.nLockTime = std::max(0, chainActive.Height() - 10);
 
+    // Set nBlockExpiry to 20 blocks (default) past current blockheight
+    txNew.nBlockExpiry = chainActive.Height() + nBlockExpiry;
```

Comment:
Should rename `nBlockExpiry` to not be semantically confused with `txNew.nBlockExpiry`.

Reply:
`blockExpiry`?